src/Flood/DatabaseBackend.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * Definition of Drupal\service_container\Flood\DatabaseBackend.
+ */
+
+namespace Drupal\service_container\Flood;
+
+use Drupal\service_container\Legacy\Drupal7;
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Flood\FloodInterface;
+
+/**
+ * Defines the database flood backend. This is the default Drupal backend.
```

Comment:
Lets add @codeCoverageIgnore here, I agree with @dawehner that unit test coverage is not needed for this class.

This is fine to go in without unit tests, because we would just test the lines literally - except for Connection, but that is way too much effort for way too less win.